pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/constants/ConfigFile.java

```diff
@@ -139,6 +139,9 @@ public class ConfigFile {
     public static final String AUTH0_IP_ALLOW_LIST = "auth0.ipAllowList";
     public static final String PREFERRED_SOURCE_IP_HEADER = "preferredSourceIPHeader";
     public static final String EMAIL_PATTERN_DENY_LIST = "emailPatternDenyList";
+    public static final String ALLOW_OS1_USER_CREATION = "allowOsteo1UserCreation";
+    public static final String OS1_USER_DEFAULT_PWD = "osteo1UserDefaultPwd";
+    public static final String OSTEO_AUTH0_CLIENT_ID = "osteoClientId";
```

Comment:
Do we have a new client just for this route?  If we're re-using an existing client, I think we can join `umbrella_study -> auth0_tenant -> client` and hit the db directly, assuming the study_guid is stable across non-prod.

Reply:
unless I am missing something, tenant is same for cmi studies and I needed osteo angular client to register as osteo user.